src/main/java/school/hei/patrimoine/google/GoogleApi.java

```diff
@@ -51,10 +54,20 @@ public boolean isTokenExpired() {
     new File(TOKENS_DIRECTORY_PATH).mkdirs();
   }
 
-  private static final List<String> SCOPES = List.of(DOCUMENTS_READONLY);
+  private static final List<String> SCOPES = List.of(DOCUMENTS_READONLY, DRIVE_READONLY);
   private static final String CREDENTIALS_FILE_PATH =
       System.getProperty("user.home") + "/.patrimoine/google/client.json";
 
+  public static final String DOWNLOADS_DIRECTORY_PATH =
+      System.getProperty("user.home") + "/Downloads/drive";
```

Comment:
to ensure the directories you will create allow READ and WRITE, use /temp.
There's a dedicated Java utility which gets temp dir of system

Reply:
done